### PR TITLE
tidb-in-kubernetes: recovery after accidental deletion

### PR DESCRIPTION
**Pull request by weekface**
Preview: https://github.com/weekface/docs-cn/blob/recovery-after-accidental-deletion/tidb-in-kubernetes/recovery-after-accidental-deletion.md

### CODE REVIEW

File: tidb-in-kubernetes/recovery-after-accidental-deletion.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: 集群意外删除后恢复
+category: troubleshooting
+---
+
+# 集群意外删除后恢复
+
+TiDB Operator 使用 PV/PVC 来存储持久化的数据，如果不小心使用 helm delete 意外删除集群后，PV/PVC 对象以及数据都会保留下来，以最大程度保证数据安全。
```
Comment (lilin90) [inline tidb-in-kubernetes/recovery-after-accidental-deletion.md]:
```suggestion
TiDB Operator 使用 PV/PVC 来存储持久化的数据，如果不小心使用 `helm delete` 意外删除了集群，PV/PVC 对象以及数据都会保留下来，以最大程度保证数据安全。
```

File: tidb-in-kubernetes/recovery-after-accidental-deletion.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+此时集群恢复的办法就是使用 helm install 命令来创建一个同名的集群，之前保留下来未被删除的 PV/PVC 以及数据会被复用：
```
Comment (lilin90) [inline tidb-in-kubernetes/recovery-after-accidental-deletion.md]:
```suggestion
此时集群恢复的办法就是使用 `helm install` 命令来创建一个同名的集群，之前保留下来未被删除的 PV/PVC 以及数据会被复用：
```

File: tidb-in-kubernetes/recovery-after-accidental-deletion.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: 集群意外删除后恢复
+category: troubleshooting
+---
+
+# 集群意外删除后恢复
```
Comment (lilin90) [inline tidb-in-kubernetes/recovery-after-accidental-deletion.md]: We can directly add this section to https://github.com/pingcap/docs-cn/blob/master/dev/how-to/maintain/backup-and-restore.md and add a subheading at the bottom. Could you please move it?

File: tidb-in-kubernetes/recovery-after-accidental-deletion.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (lilin90) [inline tidb-in-kubernetes/recovery-after-accidental-deletion.md]: Please rename this file to `tidb-in-kubernetes.md` and move it to `/dev/how-to/troubleshoot/tidb-in-kubernetes.md`.

File: dev/how-to/troubleshoot/tidb-in-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# 集群意外删除后恢复
+
+TiDB Operator 使用 PV(Persistent Volume)/PVC(Persistent Volume Claim) 来存储持久化的数据，如果不小心使用 `helm delete` 意外删除了集群，PV/PVC 对象以及数据都会保留下来，以最大程度保证数据安全。
+
+此时集群恢复的办法就是使用 `helm install` 命令来创建一个同名的集群，之前保留下来未被删除的 PV/PVC 以及数据会被复用：
+
+```shell
+helm install charts/tidb-cluster -n ${releaseName} --namespace=${namespace}
```
Comment (aylei) [inline dev/how-to/troubleshoot/tidb-in-kubernetes.md]:
```suggestion
helm install pingcap/tidb-cluster -n ${releaseName} --namespace=${namespace} --version=v1.0.0-beta.3 -f values.yaml
```

Comment (weekface) [reply dev/how-to/troubleshoot/tidb-in-kubernetes.md]: tidb-cluster 是放在 charts 目录下的

Comment (aylei) [reply dev/how-to/troubleshoot/tidb-in-kubernetes.md]: 海龙那边修改成使用 helm repository 的方式进行部署了，`pingcap/tidb-cluster` 是 <repo_name>/<chart_name>

**Comment by yikeke** [comment]
**Note:** I have updated all code blocks in this PR according to our new code block format rules, please see [Update code block format to better support copy](https://internal.pingcap.net/jira/browse/ICM-95) for details. 

PTAL. @weekface @lilin90 

And if this is a newly added file, we should add it into TOC.md.

**Comment by lilin90** [comment]
@yikeke I'll add it to TOC.md later.

**Pull request merged**